tests/testthat/test_get_panel.R

```diff
@@ -1,14 +1,16 @@
 context("get_panel")
 
 test_that("get panel", {
-  p <- ggplot(iris, aes(Sepal.Length, Sepal.Width, color=Species)) + geom_point()
+  p <- ggplot(iris, aes(Sepal.Length, Sepal.Width, color = Species)) + geom_point()
```

Comment:
See above about made-up datasets. I realize that I wrote this test initially. Some of the cowplot code is quite old and I didn't necessarily know what I was doing then.